GoogleSignIn/Sources/GIDSignIn_Private.h

```diff
@@ -29,12 +29,18 @@ NS_ASSUME_NONNULL_BEGIN
 @class GIDGoogleUser;
 @class GIDSignInInternalOptions;
 @class GTMKeychainStore;
+@class GIDVerifiedAccountDetailResult;
 
 /// Represents a completion block that takes a `GIDSignInResult` on success or an error if the
 /// operation was unsuccessful.
 typedef void (^GIDSignInCompletion)(GIDSignInResult *_Nullable signInResult,
                                     NSError *_Nullable error);
 
+/// Represents a completion block that takes a `GIDVerifiedAccountDetailResult` on success or an
+/// error if the operation was unsuccessful.
+typedef void (^GIDVerifyCompletion)(GIDVerifiedAccountDetailResult *_Nullable verifiedResult,
```

Comment:
We are only supporting iOS for this feature so far. Please check for iOS here and throughout all of your changes for this feature.

Reply:
Done.